src/ReplaceImage.js

```diff
@@ -41,15 +40,21 @@ function ReplaceImage(ref, selector, steps, options) {
     else make(url);
 
     function make(url) {
+
+      var tempSequencer = ImageSequencer({ui: false});
```

Comment:
What is temp sequencer used for, here?

Reply:
This leads back to the origin of this function - i believe it just boots up a sequencer with no UI, to run the replacement, from https://github.com/publiclab/image-sequencer/issues/32

Reply:
Hmm, is it required?